Lib/test/test_ast/test_ast.py

```diff
@@ -2277,6 +2277,7 @@ def test_values(self):
         for value in values:
             with self.subTest(value=value):
                 result = self.compile_constant(value)
+                self.assertIs(result, value)
```

Comment:
It looks like `test_ast.test_ast.ConstantTests.test_values` is failing here.

Reply:
Ooops! I didn't mean to commit that.